packages/react-native-ui/src/Layout.tsx

```diff
@@ -133,7 +137,29 @@ export const Layout = ({
 
   return (
     <View style={{ flex: 1, paddingTop: insets.top, backgroundColor: theme.background.content }}>
-      <View style={{ flex: 1, overflow: 'hidden' }}>{children}</View>
+      <View style={{ flex: 1, overflow: 'hidden' }}>
+        {children}
+
+        <TouchableOpacity
+          style={{
+            position: 'absolute',
+            bottom: uiHidden ? 56 + insets.bottom : 16,
+            right: 16,
+            backgroundColor: 'white',
+            padding: 4,
+            borderRadius: 4,
+            borderWidth: 1,
+            borderColor: theme.appBorderColor,
+          }}
+          onPress={() => setUiHidden(!uiHidden)}
+        >
+          {uiHidden ? (
+            <CloseFullscreenIcon color={theme.color.mediumdark} />
+          ) : (
+            <FullscreenIcon color={theme.color.mediumdark} />
```

Comment:
Probably it's better to use `theme.background.content`. 
Otherwise it does not respeect browsers darkmode, right?